### ISSUE TITLE
Incorrect remaining time with home assistant timers

### ISSUE THREAD

**Issue by Xe138** [rianadon/timer-bar-card#14]
This card is not working properly with the Home Assistant built-in timer integration: https://www.home-assistant.io/integrations/timer/

The card incorrectly calculates the remaining time, as shown below.

Timer configuration
```
timer:
  baby_feeding:
    duration: 00:00:30
  baby_diaper:
    duration: 00:00:30
```

Card configuration
```
type: custom:timer-bar-card
debug: true
duration:
  attribute: duration
end_time:
  attribute: finishes_at
entities:
  - timer.baby_feeding
  - timer.baby_diaper
```
Result
![image](https://user-images.githubusercontent.com/50377201/127017974-b67d9479-9295-452c-9b1f-b22da5be62bb.png)

**Comment by rianadon** [comment]
Thanks for the report! This is very strange; I cannot reproduce with your configuration.

Even with `duration` and `end_time` specified (which you don't need), the card will recognize it's a timer and use the same algorithm Home Assistant uses for computing timer time remaining: use the `remaining` attribute as the timer duration, and subtract how long has past since the timer was activated (current time minus the `last_changed` property).

I'd appreciate if you could check the following:
1. Do these timers work correctly with the native Home Assistant timer display?
2. What does the developer tools state show after you turn on the timer?
3. Is the last changed time accurate? (e.g. I started my recording at 11:15:42 and the last changed is 11:15:44 which is right)

You can search for the timer to find the last changed time like I show below:

https://user-images.githubusercontent.com/18223213/127038385-65f13b2e-920d-43d8-9656-abf44f444af2.mov

**Comment by Xe138** [comment]
I had added the "duration" and "end_time" parameters while trying to troubleshoot.

> I'd appreciate if you could check the following:
> 
> 1. Do these timers work correctly with the native Home Assistant timer display?
> 2. What does the developer tools state show after you turn on the timer?
> 3. Is the last changed time accurate? (e.g. I started my recording at 11:15:42 and the last changed is 11:15:44 which is right)

1.  Actually they do not.  I get the same incorrect time calculations on the basic entities card.  This should have been a 1 minute timer:

![image](https://user-images.githubusercontent.com/50377201/127058873-3c7aa27b-c693-4c40-ade6-e7fc72b22c71.png)

The timers still behave correctly (i.e. they complete after the correct amount of time has elapsed).

2. Here's the state information from the developer tools after starting one of the timers:

![image](https://user-images.githubusercontent.com/50377201/127059178-f4d2f14c-0ba2-4191-a65c-78483d76f980.png)

3. The "last changed" time appears to be 1 minute and 10 seconds ahead of the time on the system that I am accessing Home Assistant from.  This matches the difference in the timer calculation. The "finishes_at" time appears to be calculated correctly from the last changed time.

I tested with my phone and the timers behave correctly.  I suspect this is because the browser viewing the page is comparing the "finishes_at" time to the system time.  If the system time doesn't match the server time, the remaining time is calculated incorrectly.  Since this problem seems to be a result of the native timer implementation, I'm not sure if this can be fixed here.
